tf-modules/cumulus_distribution/variables.tf

```diff
@@ -37,9 +37,34 @@ variable "api_gateway_stage" {
   description = "The API Gateway stage name for the distribution App"
 }
 
+variable "cmr_acl_based_credentials" {
+  type = bool
+  default = false
+  description = "Option to enable/disable user based CMR ACLs to derive permission for s3 credential access tokens"
+}
+
+variable "cmr_environment" {
+  description = "The CMR environment to access"
+  type        = string
+  default     = null
+}
+
+variable "cmr_provider" {
+  description = "The provider used to search CMR ACLs"
+  type        = string
+  default     = null
+}
+
+variable "deploy_s3_credentials_endpoint" {
```

Comment:
Is this used?  With the current implementation, we always have the s3Credential endpoint in cumulus_distribution

Reply:
Gotcha. Removed.